run.js

```diff
@@ -73,10 +74,24 @@ async function main () {
 async function parseExperimentsConfig (config) {
   const experiments = []
 
+  for (const expConfig of config.driveGet) {
+    const params = {
+      nrFiles: expConfig.nrFiles,
+      fileByteSize: expConfig.fileByteSize
+    }
+
+    experiments.push({
+      experimentClass: DriveGetExperiment,
+      params,
+      name: 'drive_get',
+      description: 'Get files from Hyperdrive'
+    })
+  }
+
   for (const expConfig of config.download) {
     const params = {
-      nrBlocks: expConfig.nrBlocks,
-      blockByteSize: expConfig.blockByteSize
+      nrFiles: expConfig.nrFiles,
```

Comment:
You should not change the parameter parsing of the existing experiments (this caused the tests to break, see CI failure)

Reply:
sorry, my bad.